Update Readme.md broken links, Added other documentation files

chaitak-gorai opened the pull request:
## Description
Documentation files like CONTRIBUTING.md and CODE_OF_CONDUCT.md are important for any open-source project. They provide guidance to the contributors and maintainers on how to contribute and behave in the community and ensure that the project is welcoming to new contributors and sets clear expectations for everyone involved.

## Changes Made
- Added `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md` files
- Updated broken links in `README.md`
- Added `CHANGELOG.md` generated by `generate_changelog.py` script and called as a `pre-push` git hook.

## Issue Resolved
- Resolves #2

Maintainers @muarachmann  please review my PR and suggest me some changes if needed.
Thanks for considering this pull request. 😊😊

chaitak-gorai committed:
Create CODE_OF_CONDUCT.md

chaitak-gorai committed:
Create CONTRIBUTING.md

chaitak-gorai committed:
🔊 Added CHANGELOG.md

chaitak-gorai committed:
Update README.md

muarachmann commented:
thanks will take a look at this. Also do not wait on my approval before moving to the next tasks. You can submit as many PR as possible

chaitak-gorai commented:
> thanks will take a look at this. Also do not wait on my approval before moving to the next tasks. You can submit as many PR as possible

Sure and thanks :)

muarachmann commented:
Hi @chaitak-gorai this is cool but for now we won't want to generate the change logs as thus. We can manually have the change logs or consider this in the future. Can you adjust that?

chaitak-gorai commented:
> Hi @chaitak-gorai this is cool but for now we won't want to generate the change logs as thus. We can manually have the change logs or consider this in the future. Can you adjust that?

Sure, I will do the necessary changes.

chaitak-gorai committed:
Removed CHANGELOG.md

chaitak-gorai committed:
Removed Changelog links from README

chaitak-gorai commented:
## Updated 
Based on the changes you asked for, I have removed the CHANGELOG and fixed the links in the README.
## Changes Made
- Removed `CHANGELOG.md` , `generate_changelog.py`  and `pre-push` git hook.
- Updated  links in `README.md`

@muarachmann please have a look. Please let me know if any more changes are needed.
Thank You.

muarachmann commented on README.md:
```diff
@@ -26,12 +26,11 @@ We are collaborating closely with the [LibreHealth Project](http://librehealth.i
-9. [Credits](#credits)
-10. [License](#license)
+5. [Testing](#testing)
+6. [Contributing](#contributing)
```
Nice @chaitak-gorai , can this readme involve the tech stack we are using? at least for a start

chaitak-gorai replied:
Sure @muarachmann ,
Shall I pick up the tech stacks mentioned in the about section of the repository?

muarachmann commented:
Yes and links to them is necessary

On Fri, 31 Mar 2023 at 09:42 Chaitak Gorai ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/LibreHealthIO/lh-ehr-laravel/pull/28#discussion_r1154199886>
> :
>
> > @@ -26,12 +26,11 @@ We are collaborating closely with the [LibreHealth Project](http://librehealth.i
>  2. [Installation](#installation)
>  3. [Usage](#usage)
>  4. [Documentation](#documentation)
> -5. [Change log](#change-log)
> -6. [Testing](#testing)
> -7. [Contributing](#contributing)
> -8. [Security](#security)
> -9. [Credits](#credits)
> -10. [License](#license)
> +5. [Testing](#testing)
> +6. [Contributing](#contributing)
>
> Sure,..
> Shall I pick up the tech stacks mentioned in the about section of the
> repository?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/LibreHealthIO/lh-ehr-laravel/pull/28#discussion_r1154199886>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD4X4XSW6AXQXKIZUCV76YDW62KI3ANCNFSM6AAAAAAWI47T3E>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

chaitak-gorai committed:
Added Tech Stack in README

chaitak-gorai commented:
Sure @muarachmann ,
I have added the tech stack section in the README.md and also linked the official website or docs of these.

muarachmann commented:
LGTM We can always improve this. Nice work @chaitak-gorai
